src/components/planner/sidebar/SessionController.tsx

```diff
@@ -9,8 +10,8 @@ const SessionController = () => {
   return (
     <div className="flex w-full gap-1">
       <CoursePicker />
-      {/* <CollaborativeSession /> */}
       <Refresh />
+      <CollaborativeSession />
```

Comment:
Currently, because there is no collaborative sessions implemented yet, this should be inside an if seeing if the PROD env variable is 1 or 0.

```js
{import.meta.env.VITE_APP_PROD === 0 && <CollaborativeSession />}
```

instead of just 

```js
<CollaborativeSession />
```

Reply:
for some reason import.meta.env.VITE_APP_PROD is a string so I will use == for now

Reply:
What about creating a <DevMode> component to encapsulate that logic and use it every time we want to hide things in production?

usage example:
```
<DevMode> 
    <CollaborativeSession />
</DevMode>
```

Reply:
I think it's a great idea! It looks much cleaner, and as you pointed out, this change also makes the code more reusable.